src/material-experimental/mdc-radio/radio.ts

```diff
@@ -151,11 +159,15 @@ export class MatRadioButton extends _MatRadioButtonBase implements AfterViewInit
     this._radioFoundation.destroy();
   }
 
-  private _setClass(cssClass: string, active: boolean) {
+  setClass(cssClass: string, active: boolean) {
```

Comment:
This should be prefixed with an underscore, otherwise it's implied that it's a public API.